packages/core/src/utils/JsonTransformer.ts

```diff
@@ -25,8 +51,12 @@ export class JsonTransformer {
     return JSON.stringify(this.toJSON(classInstance))
   }
 
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  public static deserialize<T>(jsonString: string, Class: { new (...args: any[]): T }): T {
-    return this.fromJSON(JSON.parse(jsonString), Class)
+  public static deserialize<T>(
+    jsonString: string,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    Class: { new (...args: any[]): T },
+    { validate = true }: Validate = {}
```

Comment:
Same here with the default object assignment.

Reply:
same as above